site/stylesheets/index.scss

```diff
@@ -80,6 +81,8 @@ h1 {
 }
 
 .c-infinity__block {
+	margin: 0 15px;
+	padding: 0 15px;
```

Comment:
These two lines should not be in this pull request... But since there is already a new one with the fix for the jumps, its ok

Reply:
True. 😴